CMakeLists.txt

```diff
@@ -32,21 +32,12 @@ include(GNUInstallDirs)
 
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 
-if (NOT TARGET Vulkan::Headers)
-    find_package(VulkanHeaders CONFIG REQUIRED QUIET)
-endif()
-
-target_compile_definitions(Vulkan::Headers INTERFACE VK_ENABLE_BETA_EXTENSIONS) # Enable beta Vulkan extensions
```

Comment:
This can potentially cause problems for users. I already fixed this in VVL might as well fix this while I'm here.